src/util/chains.ts

```diff
@@ -65,6 +68,8 @@ export const NETWORKS_WITH_SAME_UNISWAP_ADDRESSES = [
   ChainId.ROPSTEN,
   ChainId.RINKEBY,
   ChainId.GÖRLI,
+  // not sure if this is correct?
+  ChainId.SEPOLIA,
```

Comment:
Don't think this is true, this would mean that V3PoolFactory / NFTPositionManager / etc. are deployed on sepolia to the same addresses as mainnet, gorli, etc.

Reply:
ah ty, removed